.github/workflows/publish_pypi.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: python -m pip install cibuildwheel==1.9.0
```

Comment:
We need to bump this up to latest version (v2.12.3) in both steps (Windows and non Windows)

Reply:
done, see action below